src/app/admin/components/user-card/user-card.component.scss

```
@@ -4,6 +4,30 @@
     box-shadow: 0px 0px 12px 1px rgba(0, 0, 0, 0.05);
     width: 185px;
     border-radius: 5px;
+    transition: transform .2s;
```

can we add the easing property here? 
```suggestion
    transition: transform 0.2s ease;
```